context_helper/context.go

```diff
@@ -0,0 +1,10 @@
+package context_helper
+
+import (
+	"context"
+	"time"
+)
+
+func GetContext() (context.Context, context.CancelFunc) {
+	return context.WithTimeout(context.Background(), 1000*time.Second)
```

Comment:
When I've ran it locally I've got a TO from the realistic time (such as 100 ms) since it takes time for the request to  reach from my computer. I'll try 1s